packages/cli/src/PublicApi/v1/handlers/users/spec/paths/users.yml

```diff
@@ -18,3 +18,53 @@ get:
             $ref: '../schemas/userList.yml'
     '401':
       $ref: '../../../../shared/spec/responses/unauthorized.yml'
+post:
```

Comment:
We should look into migrating the public api to a similar pattern using decorators and DI, and then generate the openapi spec file from the decorator metadata instead (if possible).

Reply:
100%! For context, we discussed internally and are aware of the all the problems of the Public API, but Enterprise customers need this so we're expanding it as it is for now.